### DISCUSSION TITLE
What is the minimum cache length I can specify without request rejection in Remote Config?

### DISCUSSION THREAD

**Discussion by SoohyungJung** [invertase/react-native-firebase#6410] [Q&A]
Hello,

I have been trying to figure out whether I can utilize Remote Config to set maintenance mode in a React Native project without caching issues.

> To bypass caching fully, you can pass a value of 0. Be warned Firebase may start to reject your requests if values are requested too frequently.

According to the [Remote Config documentation](https://rnfirebase.io/remote-config/usage#caching) like above, it seems like I am able to specify my own cache length as 0 but it can possibly cause request rejections.

Then, I wonder if it is okay to set a value other than 0 such as 30000 to the `minimumFetchIntervalMillis` property to avoid rejections, or is there any certain minimum cache length to do so?

Thank you in advance.

Soo

**Comment by mikehardy** [comment]
Hi there! This is best asked on stackoverflow with firebase tags (to attract attention of firebase team members from google) as it is not react-native specific, so we won't have authoritative answers here. In general though, this is just how long your app - on each mobile device - will consider a remote value cached locally as "authoritative". That's it. If the local value has been feetched more than minimumFetchIntervalMillis milliseconds ago, it will discard local value and refetch from server. The server may throttle your requests. Zero means never cache and that's probably not what you want.

**Reply by SoohyungJung** [reply]
Alright I will ask there as well. If I get to know answers hopefully from firebase team, I will share them here so that we all can be clear about this.

Thank you for your time!
